caffe2/core/tensor.h

```diff
@@ -433,6 +433,10 @@ class TORCH_API Tensor final {
     return impl_->sym_sizes();
   }
 
+  inline c10::SymInt sym_numel() const {
+    return impl_->sym_numel();
+  }
```

Comment:
There's no reason to add the method here (same with the others). Legacy Caffe2 interface